### ISSUE TITLE
Read Avro document row-by-row

### ISSUE THREAD

**Issue by thorhj** [AdrianStrugala/AvroConvert#18]
I am working with a huge `.avro` file that contains many rows of data. I want to process this row-by-row (i.e. entry-by-entry) without reading everything into memory. Is this possible using `AvroConvert`?

Using `Apache.Avro`, I believe I can do this like so:

```csharp
var reader = DataFileReader<T>.OpenReader(stream);
while (reader.HasNext())
{
    var item = reader.Next();
    // process item
}
```

I can understand if this is not supported by this library, but I would love to use the automatic (case-insensitive) schema-generation provided by `AvroConvert`.

**Comment by AdrianStrugala** [comment]
At this moment this kind of deserialization is not supported by AvroConvert.
I'll have to analyze this feature.

Thanks for your idea!

**Comment by AdrianStrugala** [comment]
The feature was implemented in version 2.6.0

Syntax is as follows:
```csharp
using (var reader = AvroConvert.OpenDeserializer<User>(new MemoryStream(_avroBytes)))
{
    while (reader.HasNext())
    {
        var item = reader.ReadNext();
        // process item
    }
}
```
